tgui/packages/tgui/interfaces/PreferencesMenu/AntagsPage.tsx

```diff
@@ -47,9 +47,8 @@ const AntagSelection = (props: { antagonists: Antagonist[]; name: string }) => {
   const { act, data } = useBackend<PreferencesMenuData>();
   const className = 'PreferencesMenu__Antags__antagSelection';
 
-  const [predictedState, setPredictedState] = useLocalState(
-    'AntagSelection_predictedState',
-    new Set(data.selected_antags),
+  const [predictedState, setPredictedState] = useState(
+    new Set(data.selected_antags)
```

Comment:
this makes me anxious. perhaps this should be made constant beforehand?

Reply:
@Mothblocks

Reply:
![emBYPzUa8T](https://github.com/tgstation/tgstation/assets/5195984/f4db8a09-4a99-4d6a-9b75-d6f0ccffde77)